r/tests/testthat/test-type.R

```diff
@@ -94,7 +94,7 @@ test_that("infer_type() can infer blob type", {
 
   expect_equal(infer_type(blob::blob()), binary())
 
-  big_ish_raw <- raw(2 ^ 20)
+  big_ish_raw <- raw(2^20)
```

Comment:
Apologies, I ran `make style-all` and so we have some other style fixes around. Should I revert these and add them in a separate PR?

Reply:
It doesn't look like there are that many, all good.